maven-core/src/main/java/org/apache/maven/project/MavenProject.java

```diff
@@ -708,6 +708,9 @@ public List<Plugin> getBuildPlugins() {
     }
 
     public List<String> getModules() {
```

Comment:
Deprecate it? Or maybe does not matter, as in API 4 plugins have no reach to this class?

Reply:
Yes, I was assuming this change targets 4.x api and plugins, so I would not care about the v3 api.